cmd/controller-manager/app/controller_manager.go

```diff
@@ -342,3 +349,27 @@ func StartControllers(s *options.ControllerManagerServer,
 
 	select {}
 }
+
+var _ healthz.HealthzChecker = checkAPIAvailableResources{}
```

Comment:
Do you want me to take it out or do you want a comment? I want to make sure we know what it is. We do have the comment, I guess.